pkg/api/steve/disallow/disallow.go

```diff
@@ -27,13 +27,19 @@ var (
 		"templateversions":          true,
 		"tokens":                    true,
 	}
+	mgmtWhiteList = map[string]bool{
```

Comment:
Somewhat of a not, but I think it's best to name it `allowAll` to match the existing maps naming style and move it to the top of the var block definition.